scripts/tests/test_validate_docstrings.py

```diff
@@ -229,6 +229,24 @@ def good_imports(self):
         """
         pass
 
+    def does_nothing(self, x):
+        """
+        Do nothing and always return none.
+
+        But in other cases it might also
+        return nothing at all.
+
+        Parameters
+        ----------
+        x : int
+            Useless parameter.
+        """
+        var_with_return_on_the_name = random.random()
```

Comment:
I see your point, but in practice I don't think we need to return in both, you're just using regex. `if True` and avoid random values should be enough.

Also, you may want to review how the modulo operator performs with a float, your condition will never return False anyway.

Reply:
I removed random values.